packages/components/components/elvis-pagination/src/react/elvia-pagination.test.tsx

```diff
@@ -79,42 +79,42 @@ describe('Elvis Pagination', () => {
     });
 
     it('should have hidden left arrow', () => {
-      const previousButton = screen.queryByRole('button', { name: /forrige side/i });
+      const previousButton = screen.queryByRole('button', { name: /previous page/i });
       expect(previousButton).not.toBeInTheDocument();
     });
 
     it('should have visible right arrow', () => {
-      const nextButton = screen.queryByRole('button', { name: /neste side/i });
+      const nextButton = screen.queryByRole('button', { name: /next page/i });
       expect(nextButton).toBeInTheDocument();
     });
 
     it('should have visible left arrow after clicking right arrow', async () => {
       const user = userEvent.setup();
-      const nextButton = screen.getByRole('button', { name: /neste side/i });
+      const nextButton = screen.getByRole('button', { name: /next page/i });
 
       await user.click(nextButton);
-      const previousButton = screen.getByRole('button', { name: /forrige side/i });
+      const previousButton = screen.getByRole('button', { name: /previous page/i });
       expect(previousButton).toBeVisible();
     });
 
     it('should have hidden right arrow after clicking last paginator number', async () => {
       const user = userEvent.setup();
 
-      const lastPageButton = screen.getByRole('button', { name: /velg side 10/i });
+      const lastPageButton = screen.getByRole('button', { name: /Select page 10/i });
```

Comment:
Nit:
```suggestion
      const lastPageButton = screen.getByRole('button', { name: /select page 10/i });
```
just for consistency 🤷🏽